docs/dev/README.md

```diff
@@ -34,10 +34,17 @@ If this is your first time using zkSync, we recommend that you kick off at the b
 - [Status](./troubleshooting/status.md) - Get updates on things we are currently working on.
 - [Known Issues](./troubleshooting/known-issues.md) - Get answers to common issues you may find.
 
-### Developer Tools
+### Tools and SDKs
 
 - [zkSync 2.0 Portal](https://portal.zksync.io) - Explore Wallet, Bridge and Faucet features.
 - [Block explorer](../api/tools/block-explorer/) - Search for real-time and historical information about blocks, transactions, addresses, and more on zkSync block explorer.
+- [Javascript SDK](../api/js/) - Get the neccessary javascript toolkits to build dAPPs on zkSync.
```

Comment:
javascript toolkits doesn't make too much sense.

And maybe we should simply and have the same sentence for all SDKs... "Explore all the <language> methods and functions required to build on zkSync"

Reply:
how about "Extending the capabilities of Ethers, our Javascript SDK contains specific classes and methods required to build on zkSync 2.0".  Also add, "zkSync 2.0" to the other SDKs.